SQL/0000-00-01-Permission.sql

```diff
@@ -111,7 +111,20 @@ INSERT INTO `permissions` VALUES
     (49,'data_release_upload', 'Data Release: Upload file', 2),
     (50,'data_release_edit_file_access', 'Data Release: Grant other users view-file permissions', 2),
     (51,'instrument_manager_read', 'Instrument Manager: View module', 2),
-    (52,'instrument_manager_write', 'Instrument Manager: Install new instruments via file upload', 2);
+    (52,'instrument_manager_write', 'Instrument Manager: Install new instruments via file upload', 2),
+    (53, 'biobank_specimen_view', 'Biobank: View Specimen Data', 2),                
+    (54, 'biobank_specimen_create', 'Biobank: Create Specimen Data', 2),            
+    (55, 'biobank_specimen_update', 'Biobank: Update Specimen Data', 2),            
+    (56, 'biobank_specimen_alter', 'Biobank: Alter Specimen Data', 2),              
+    (57, 'biobank_container_view', 'Biobank: View Container Data', 2),              
+    (58, 'biobank_container_create', 'Biobank: Create Container Data', 2),          
+    (59, 'biobank_container_update', 'Biobank: Update Container Data', 2),          
```

Comment:
Should there be alter permissions for containers and pools?

Reply:
From discussing it with Rida I don't think it seemed necessary at the time, but it's highly likely that it could come up in the future. 

@ridz1208 thoughts?

Reply:
You are correct, features for `alter` have not yet been implemented. I can remove the permission until it has a use, if that's agreeable to everyone.

@johnsaigle @ridz1208